src/main/scala/chisel3/Driver.scala

```diff
@@ -127,6 +127,20 @@ object Driver extends BackendCompilationUtilities {
     f
   }
 
+  /**
+    * Emit the annotations of a circuit
+    *
+    * @param ir The circuit containing annotations to be emitted
+    * @param optName An optional filename (will use s"${ir.name}.json" otherwise)
+    */
+  def dumpAnnotations(ir: Circuit, optName: Option[File]): File = {
+    val f = optName.getOrElse(new File(ir.name + ".json"))
```

Comment:
I generally prefer `.anno.json`, but it's not a big deal

Reply:
I generally prefer it, too! Good catch.